Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -378,14 +379,14 @@ void FullscreenManager::exitFullscreen(RefPtr<DeferredPromise>&& promise)
     });
 }
 
-void FullscreenManager::finishExitFullscreen(Document& currentDocument, ExitMode mode)
+void FullscreenManager::finishExitFullscreen(const Ref<Document>& currentDocument, ExitMode mode)
```

Comment:
cc @rniwa for his opinion on this.

Reply:
I heard back for Ryosuke, the old style is still preferred. Please omit this change.